README.md

```diff
@@ -6,4 +6,6 @@ simulations by automating commonly performed tasks.
 
 See [documentation](https://e3sm-project.github.io/zppy) for more details.
 
-Run `python -m unittest tests/*.py` to run all tests.
\ No newline at end of file
+Run `python -m unittest discover -s tests -p "test_*.py` to run all tests.
```

Comment:
Updated commends to reflect new `tests` dir structure

Reply:
Add end quote: `python -m unittest discover -s tests -p "test_*.py"`